sdk/identity/azure-identity/README.md

```diff
@@ -191,6 +191,8 @@ The [Managed identity authentication](https://learn.microsoft.com/azure/active-d
 - [Azure Virtual Machines](https://learn.microsoft.com/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token)
 - [Azure Virtual Machines Scale Sets](https://learn.microsoft.com/azure/active-directory/managed-identities-azure-resources/qs-configure-powershell-windows-vmss)
 
+**Note:** Use `azure-identity` version `1.7.0` or later to utilize in-memory token caching support for managed identity authentication.
```

Comment:
The "Azure App Service and Azure Functions" link above should use `java` instead of `dotnet` in the querystring.

Also, let's link to the token caching doc from this sentence.

```suggestion
**Note:** Use `azure-identity` version `1.7.0` or later to utilize [token caching](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/identity/azure-identity/TOKEN_CACHING.md) support for managed identity authentication.
```